Makefile

```diff
@@ -241,8 +244,7 @@ changelog-release:
 
 .PHONY: changelog-unreleased
 changelog-unreleased: ## show the current changelog that will be produced on the next release (note: requires GITHUB_TOKEN set)
-	@docker run -it --rm  \
-		-v "$(shell pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator \
+	@github_changelog_generator \
```

Comment:
the `changelog-unreleased` is a target meant for local developers to be able to (optionally) use to see what the changelog would be before cutting a release. By leveraging the docker image dev's don't need to install the right version of ruby, install the gem, and get pathing correct. With the new change that would be required.

(The `changelog-release` target is OK, since this env is setup with `ci-bootstrap-mac`)

Reply:
Sounds good, reverting this.